app/code/core/Mage/Adminhtml/Block/Sales/Items/Abstract.php

```diff
@@ -449,8 +449,11 @@ public function canEditQty()
         ) {
             return false;
         }
-        if ($this->getOrder()->getPayment()->canCapture()) {
-            return $this->getOrder()->getPayment()->canCapturePartial();
+        if (($payment = $this->getOrder()->getPayment())
```

Comment:
```suggestion
        if ($payment = $this->getOrder()->getPayment()
```

I think this would be the same but in general I kinda dont like assignments inside ifs, dunno, it does seem confusing to me

Reply:
Unfortunately, that's not the same. As it was before, the brackets were necessary because of the `&&` chain. But that's not a problem. I've removed the declaration out of the `if`. :)